### PR TITLE
🚀 House: house creation test

### PR DESCRIPTION
**Pull request by shlior7**
Created a create_house fixture and a test that validates its creation

Blocked by #42

### CODE REVIEW

**Comment by svatares** [comment]
@shlior7 is this PR blocked by PR #28 ? If so, please mark it accordingly

**Comment by svatares** [comment]
@shlior7  #28 was merged, please rebase

File: house/tests/house_tests.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        )
+
+    def test_save_expense(self, generate_house):
+        generate_house.save()
+        assert generate_house in House.objects.all()
+
+    def test_delete_expense(self, generate_house):
+        generate_house.save()
+        generate_house.delete()
+        assert generate_house not in House.objects.all()
```
Comment (adiabramovitch) [inline house/tests/house_tests.py:60]: IMHO tests which validate the values of the attributes are also needed - see my examples here: 
https://github.com/redhat-beyond/Xpense/pull/42/files#r835908654

**Review by LiorKGOW — APPROVED** [review]

File: house/tests/house_tests.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            and generate_house.public == HOUSE_PUBLIC
+            and generate_house.country.name == COUNTRY
+            and generate_house.city.name == CITY
+            and generate_house.parent_profession_1 == HOUSE_PARENT_PROFESSION_1
+            and generate_house.parent_profession_2 == HOUSE_PARENT_PROFESSION_2
+            and generate_house.income == HOUSE_INCOME
+            and generate_house.children == HOUSE_CHILDREN
+        )
+
+    def test_save_expense(self, generate_house):
```
Comment (ShellyGolden) [inline house/tests/house_tests.py:53]: Need to change to test_save_house

File: house/tests/house_tests.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        generate_house.save()
+        assert generate_house in House.objects.all()
+
+    def test_delete_expense(self, generate_house):
```
Comment (ShellyGolden) [inline house/tests/house_tests.py:57]: Also here

**Review by ShellyGolden — APPROVED** [review]
looks good :)

**Review by LiorKGOW — APPROVED** [review]
LGTM

**Review by Yoavsc129 — APPROVED** [review]
Good job with the house methods and tests

**Pull request merged**